src/mapAccum.js

```diff
@@ -6,7 +6,8 @@ var _curry3 = require('./internal/_curry3');
  * function to each element of a list, passing an accumulating parameter from left to
  * right, and returning a final value of this accumulator together with the new list.
  *
- * The iterator function receives two values: *(acc, value)*
+ * The iterator function receives two arguments: *acc* and *value*, and should return
+ * a tuple *[acc, value]*.
```

Comment:
Since `,` binds more tightly than `:`, I parse this as:

    (The iterator function receives two arguments): ((*acc* and *value*), and should return a tuple *[acc, value]*).

We should replace the `:` with a `,` to ensure correct parsing:

    The iterator function receives two arguments, (*acc* and *value*), and should return a tuple *[acc, value]*.